api/models/event.js

```diff
@@ -0,0 +1,36 @@
+const { types, labels } = require('../utils/event');
+
+const associate = () => {};
+
+module.exports = (sequelize, DataTypes) => {
+  const Event = sequelize.define('Event', {
+    type: {
+      type: DataTypes.ENUM,
+      values: Object.values(types),
+      allowNull: false,
+    },
+    label: {
+      type: DataTypes.ENUM,
+      values: Object.values(labels),
+      allowNull: false,
+    },
+    modelId: {
+      type: DataTypes.INTEGER,
+    },
+    model: {
+      type: DataTypes.ENUM,
+      values: Object.keys(sequelize.models),
```

Comment:
I think we're better of implementing this as a validation that will check  `sequelize.models` at *validation* time, otherwise, this requires that the `Event` model be the last model to be created to ensure that all models are included in the validation. We could easily forget and screw this up.